pkg/minishift/hostfolder/hostfolder.go

```diff
@@ -327,6 +327,10 @@ func mountCifsHostfolder(driver drivers.Driver, hostfolder *config.HostFolder) e
 		hostfolder.Options["username"],
 		password)
 
+	if !miniutils.IsB2d(driver) {
```

Comment:
If ISO is minikube, this will fail. Checking for SELinux state or IsCentOS would be a better approach.

Reply:
feel free to jump on this one